app.rb

```diff
@@ -12,26 +12,96 @@ def initialize
     @rentals = []
   end
 
-  def user_input(text)
-    print text
-    gets.chomp
+  class CreateList
+    def user_input(text)
+      print text
+      gets.chomp
+    end
   end
 
   def list_all_books
-    if @books.empty?
+    ListBook.new.list_all_books(@books)
+  end
+
+  def list_all_people
+    ListPeople.new.list_all_people(@people)
+  end
+
+  def create_a_person
+    CreateList.new.create_a_person(@people)
+  end
+
+  def create_a_book
+    CreateList.new.create_a_book(@books)
+  end
+
+  def create_a_rental
+    CreateList.new.create_a_rental(@books, @people, @rentals)
+  end
+
+  def list_all_rentals
+    ListRental.new.list_all_rentals(@rentals, @people)
+  end
+
+  def operation(input)
+    case input
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_a_person
+    when 4
+      create_a_book
+    when 5
+      create_a_rental
+    when 6
+      list_all_rentals
+    else
+      puts 'Please choose a valid option number'
+    end
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+    loop do
+      puts '*' * 60
+      puts '
+          1- List all books
+          2- List all people
+          3- Create a new person
+          4- Create a new book entry
+          5- Create a new rental entry
+          6- List all rentals for a given person id
+          7- Quit'
+
+      input = user_input('Choose an option: ').to_i
+
+      break if input == 7
+
+      operation(input)
+    end
+  end
+end
```

Comment:
- [ ] I think you wrote your options here but as required Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit). so please refactor this to be in the main file so the main responsiblity for the `main.rb` file just to display the options(or quit) by that you are following  ` Single-responsibility Principle` 🔥✨💯♻♻